cmd/tnf/generate/catalog/catalog.go

```diff
@@ -157,19 +254,28 @@ func outputTestCaseBuildingBlocks() {
 	// Sorting the map by identifier URL
 	sort.Strings(keys)
 
-	// Iterating the map by sorted identifier URL
-	for _, k := range keys {
-		fmt.Fprintf(os.Stdout, "### %s", identifier.Catalog[k].Identifier.URL)
-		fmt.Println()
-		fmt.Println("Property|Description")
-		fmt.Println("---|---")
-		fmt.Fprintf(os.Stdout, "Version|%s\n", identifier.Catalog[k].Identifier.SemanticVersion)
-		fmt.Fprintf(os.Stdout, "Description|%s\n", identifier.Catalog[k].Description)
-		fmt.Fprintf(os.Stdout, "Result Type|%s\n", identifier.Catalog[k].Type)
-		fmt.Fprintf(os.Stdout, "Intrusive|%t\n", identifier.Catalog[k].IntrusionSettings.ModifiesSystem)
-		fmt.Fprintf(os.Stdout, "Modifications Persist After Test|%t\n", identifier.Catalog[k].IntrusionSettings.ModificationIsPersistent)
-		fmt.Fprintf(os.Stdout, "Runtime Binaries Required|%s\n", cmdJoin(identifier.Catalog[k].BinaryDependencies, ", "))
-		fmt.Println()
+	catalog := createPrintableCatalogFromUrls(keys)
```

Comment:
This section is for the handlers. They are meant be used as go library when writing test cases. So I don't see a need of updating this the same way as the test cases section.

Reply:
As discussed with @jgato, we will keep this change for better readability.